src/promptix/tools/version_manager.py

```
@@ -0,0 +1,441 @@
+            try:
+                common_path = Path(os.path.commonpath([resolved_base, resolved_candidate]))
+                is_contained = common_path == resolved_base
+            except ValueError:
+                # commonpath raises ValueError if paths are on different drives (Windows)
+                is_contained = False
+            
+            if not is_contained:
+                self.print_status(f"Invalid {path_type}: path traversal detected", "error")
+                return False
+                
+            return True
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)